paper.md

```diff
@@ -0,0 +1,178 @@
+During the course learners type in the commands that are explained by the instructor on their own computer.
+This ensures that learners master the programmatic implementation of deep learning at the end of the course. 
+
+Researchers can often only free a limited amount of time (maximum 5 consecutive days), since they are so involved in their daily work.
+To accomplish this, we created a lesson that can be taught in 2 consecutive days or 4 mornings.
+
+From the feedback we gathered from students enrolled in our workshops and instructors not involved in our project we see that indeed there is a 
+need for a low-threshold lesson that lets researchers take the first steps in the field of deep learning.
+
+# Instructional design
+This lesson material was designed using the concepts from the Carpentries Curriculum Development Handbook (REFERENCE https://cdh.carpentries.org/).
+Most importantly, we used 'backward design': We started with identifying learning objectives: the core skills and concepts that learners need to acquire as a result of the lesson.
+Then, these learning objectives were used to create exercises that test whether the objectives are met.
+Eventually, the content is written in such a way that it matches what learners need to learn to successfully apply these skills.
+
+Furthermore, the lesson is built around live coding. 
+The lesson is built up of small blocks. In each block first the instructor demonstrates how to do something,
+and students follow along on their own computer. Then, the students work independently on exercises individually
+or in groups to test their skills.
+
+The lesson material is built in the new Carpentries lesson template (REFERENCE). 
+This makes the lesson material both suited as a self-study resource, a syllabus to share with students after the workshop,
+as well as lesson material for the instructor. With the 'instructor view' we even include instructor notes 
+at the level of the lesson content.
+
+The lesson is split up in a general introduction, and 3 episodes that cover 3 distinct increasingly more complex deep learning problems.
+Each of the deep learning problems is approached using the same 10-step deep learning workflow.
+By going through the deep learning cycle 3 times on different problems we hope that after the lesson learners will
+feel confident in applying this deep learning workflow to their own work.
+
+
+# Feedback
+This course was taught 7 times over the course of 3 years, both online and in person, at the Netherlands eScience Center. 
+**@Peter @Colin how often did you teach this at which institutes?**
+Apart from the core group of contributors, the workshop was also taught at 3 independent institutes, namely:
+University of Wisconson-Madison, University of Auckland, and EMBL Heidelberg.
+In general, adoption of the lesson material by the instructors not involved in the project went well.
```

Comment:
Reiterate here that feedback from instructors was used to iterate on/polish the lesson further? Or do we not need to since it was mentioned once already?

Reply:
Yes, and at least in my case, we always tried to add fixes immediately when we discovered deficiencies.

Reply:
Yes, good to add. I added 85e47dbfd1a4369dfb9729bc0147f58f25180b4d